libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -619,7 +624,14 @@ void AP_Mount_Backend::set_rctargeting_on_rcinput_change()
     const int16_t yaw_in = (yaw_ch == nullptr) ? 0 : yaw_ch->get_radio_in();
 
     // if not in RC_TARGETING or RETRACT modes then check for RC change
-    if (get_mode() != MAV_MOUNT_MODE_RC_TARGETING && get_mode() != MAV_MOUNT_MODE_RETRACT) {
+    if (last_rc_input.roll_in == 0 && last_rc_input.pitch_in == 0 && last_rc_input.yaw_in == 0)
```

Comment:
Thanks for this.  Instead of checking if roll_in, etc are zero, let's add an initialised boolean variableto the last_rc_input structure and use that instead.

Reply:
I'll make that change, then I'll need to retest. It will probably be ready tomorrow.

Reply:
Thanks Randy - I added a boolean and it is much simpler and clearer.